features/all-positions/AllPositions.tsx

```diff
@@ -87,75 +163,141 @@ const AllPositions = () => {
           </Typography>
         </Box>
         <Box pt={4}>
-          {activeSponsors && (
-            <TableContainer component={Paper}>
-              <Table>
-                <TableHead>
-                  <TableRow>
-                    <TableCell>Sponsor</TableCell>
-                    <TableCell align="right">
+          <TableContainer component={Paper}>
+            <Table>
+              <TableHead>
+                <TableRow>
+                  <TableCell>Sponsor</TableCell>
+                  <TableCell align="right">
+                    <ClickableText
+                      onClick={(e) => {
+                        setSortedColumn(SORT_FIELD.COLLATERAL);
```

Comment:
Is there any way to pull this out into a separate component or some way the logic doesn't have to be repeated like this?

Reply:
Great thinking. I did this:

```
    const SortableTableColumnHeader = ({
      children,
      sortField,
    }: SortableTableHeaderProps) => {
      return (
        <ClickableText
          onClick={(e) => {
            setSortedColumn(sortField);
            setSortDirection(
              sortedColumn !== sortField ? true : !sortDirection
            );
          }}
        >
          {children}
          {sortedColumn === sortField ? (!sortDirection ? "↑" : "↓") : ""}
        </ClickableText>
      );
    };
```

which allowed me to just simply insert:

```
<SortableTableColumnHeader
                      sortField={SORT_FIELD.COLLATERAL}
                    >
                      Collateral
                      <br />({collSymbol}){" "}
                    </SortableTableColumnHeader>
```